phpBB/phpbb/language/language_file_helper.php

```diff
@@ -67,6 +67,18 @@ public function get_available_languages() : array
 		return $available_languages;
 	}
 
+	/**
+	 * Return by given lang key its composer.json value
+	 *
+	 * @return string
+	 *
+	 */
+	public function get_lang_key_value($lang_key) : string
+	{
+		$available_languages = $this->get_available_languages();
+		return $available_languages[0][$lang_key];
```

Comment:
I am not sure, if this work fine with >1 language packs installed.

Should it not first check phpbb_user => user_lang and with that known grab the information from the composer.json to output to the specific places? 

And if user_lang (from the DB) is always the same as user_lang in the composer.json, wouldn't it be easier to only use this in the first place?

Reply:
Well, in this case DB query is not even needed as `$user->data['user_lang']` can be used. Then if you add iso as an index for `$available_languages` it can be used to match values, like `$available_languages[$user->data['user_lang']][$lang_key]`.